modules/git/git.go

```diff
@@ -105,6 +105,9 @@ func SetExecutablePath(path string) error {
 		return fmt.Errorf("installed git version %q is not supported, Gitea requires git version >= %q, %s", gitVersion.Original(), RequiredVersion, moreHint)
 	}
 
+	if err = checkGitVersionCompatibility(gitVersion); err != nil {
+		return fmt.Errorf("installed git version %s has a known compatibility issue with Gitea: %w, please downgrade (or upgrade) your git", gitVersion.String(), err)
```

Comment:
```suggestion
		return fmt.Errorf("installed git version %s has a known compatibility issue with Gitea: %w, please upgrade (or downgrade) git", gitVersion.String(), err)
```

Reply:
At the moment, downgrade seems a better solution. Because the bug also exists in 2.44.x-rc ......

Reply:
Was there confirmation of the bug? In any case, we should recommend downgrade currently, maybe add a TODO to change it to upgrade later.

Reply:
> At the moment, downgrade seems a better solution. Because the bug also exists in 2.44.x-rc ......

Yes, but only because we are talking about **this specific** bug.
However, knowing Git(ea) development, there will be more bugs in the future which are typically fixed by upgrading, which is the most common case, not downgrading.

Reply:
If we know that the patch is going to merge, then yes, we can recommend upgrade.